index.html

```diff
@@ -2863,45 +2890,28 @@ <h3>Add Proof</h3>
         <p>
 The following algorithm specifies how to add a digital proof to a document,
 which  can be used to verify the authenticity and integrity of an
-<dfn>unsecured data document</dfn>. Required inputs are an
-<a>unsecured data document</a> (<var>unsecuredDocument</var>) and
-<a>proof options</a> (<var>options</var>). The
-<a>proof options</a> MUST contain a type identifier for the
-<a>cryptographic suite</a> (<var>type</var>) and any other properties needed by
-the <a>cryptographic suite</a> type; an identifier for the
-<a>verification method</a> (<var>verificationMethod</var>) that can be used to
-verify the authenticity of the proof; an [[!XMLSCHEMA11-2]] `dateTimeStamp`
```

Comment:
An important thing we want to avoid is saying that any algorithms require private key material to be passed in opposed to an interface for performing signing. It's often the case that signals are sent to implementers that they ought to require people to pass in private key material when requiring a signer interface instead enables much better security practices (such as HSM use, etc.). These signals also sometimes create headaches where implementers choose to provide synchronous interfaces (as opposed to asynchronous ones) which do not even allow people to "squeeze in" HSM-backed signing mechanisms.

Reply:
Good point. I've tried to incorporate this in 3d6aa172a485ade9d5a5167d2a7b7ff4221064ad.